.gitignore

```diff
@@ -70,4 +70,5 @@ config
 
 # Misc
 code/common/src/config.h
-tmp
\ No newline at end of file
+tmp
+automate.py
```

Comment:
I'm also interested in this python code. Can you please send this directly to me via Mail?